CMakeLists.txt

```diff
@@ -911,21 +941,24 @@ else()
         )
 endif()        
 
-# install python modules
+# install python modules and system libs
 if (PYTHON_COPY)
-    install(DIRECTORY build_tree/${PYTHON_COPY_LIB_TARGET} 
-            DESTINATION lib
-            PATTERN *.dll EXCLUDE
-            PATTERN *.a EXCLUDE
-            PATTERN *.so EXCLUDE
-            PATTERN */dist-packages EXCLUDE
-            PATTERN *.py EXCLUDE)
+    # install all sub folders of Python
+    foreach(PY_SUB_PATH ${PY_SUB_PATHS})
+        install(DIRECTORY build_tree/${PY_SUB_PATH}/
+                DESTINATION ${PY_SUB_PATH}/ OPTIONAL)
+    endforeach(PY_PATH)
+    # copy Python's include
     install(DIRECTORY build_tree/${PYTHON_COPY_INCLUDE_TARGET} 
             DESTINATION include)
 endif()
 
-install(TARGETS    bin/flow123d  
-  RUNTIME DESTINATION bin)
+
+install(PROGRAMS    build_tree/bin/flow123d             RENAME "flow123d.bin"
```

Comment:
We should make this only for Linux platform until we deal with the windows as well.